### PR TITLE
Extracted method that 'detects push method by reg id'

### PR DESCRIPTION
**Pull request by jkblume**
Extracted method that detects device type / used push method out of send method. I will use it for logging / persisting the device type that was used by send() method.

### CODE REVIEW

File: src/push-notifications.js [hunk 1]
```diff
@@ -42,17 +72,17 @@ class PN {
 
     // Classify each pushId for corresponding device
     regIds.forEach((regId) => {
-      if (typeof regId === 'object') {
+      const pushMetdod = this.getPushMethodByRegId(regId);
```
Comment (alex-friedl) [inline src/push-notifications.js:75]: typo `pushMethod`

File: src/push-notifications.js [hunk 2]
```diff
@@ -30,6 +38,28 @@ class PN {
       });
   }
 
+  getPushMethodByRegId(regId) {
```
Comment (alex-friedl) [inline src/push-notifications.js:41]: please add test in `test/push-notifications/basic.js`

Comment (alex-friedl) [inline src/push-notifications.js:41]: please add documentation about newly exported function in `README.md`

**Review by alex-friedl — CHANGES_REQUESTED** [review]

**Comment by jkblume** [comment]
Thanks for your review and sorry for my late response. I tried to address your points and hope it matches your expectations. I am glad to hear from you.

**Review by alex-friedl — APPROVED** [review]

**Pull request merged**